devolutions-gateway/src/rdp_proxy.rs

```diff
@@ -103,20 +115,67 @@ where
     let mut client_framed = ironrdp_tokio::TokioFramed::new(client_stream);
     let mut server_framed = ironrdp_tokio::TokioFramed::new(server_stream);
 
+    let krb_server_config = if conf.debug.enable_unstable {
+        if let Some(KerberosServer {
+            realm: _,
+            users: _,
+            krbtgt_key: _,
+            max_time_skew,
+            ticket_decryption_key,
+            service_user,
+        }) = conf.debug.kerberos_server.as_ref()
+        {
+            let user = service_user.as_ref().map(|user| {
+                let DomainUser {
+                    username,
+                    password,
+                    salt: _,
+                } = user;
+                CredentialsBuffers::AuthIdentity(AuthIdentityBuffers::from_utf8(
+                    username, // The username is in the FQDN format. Thus, the domain field can be empty.
+                    "", password,
```

Comment:
suggestion: Maybe move the comment just above the line `CredentialsBuffers::AuthIdentity(AuthIdentityBuffers::from_utf8(`

Reply:
done